scripts/uncrustify.cfg

```diff
@@ -0,0 +1,160 @@
+# Uncrustify-0.67
```

Comment:
I added this file to tools/ already, shall we delete it and have it under "scripts/" instead?

Reply:
I'm fine with having it under tools (as uncrustify is a tool). amazon-freertos keeps it under tools as well.
I will remove this one then.

Reply:
Done